src/styles/blocks/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
```

Comment:
you can have another file for these variables

Reply:
is it nessesary for this project? I`m using additional variables file in the layout_catalog project, coz I`ve much more variables there.